config/config.exs

```diff
@@ -25,6 +24,17 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+# Add a base libcluster config for Epmd (erlang distribution protocol)
```

Comment:
This basically lets us test a release locally without k8s, keeping everything else equal w/ concerns to libcluster & cluster initialization.